uwblueprint/feeding-canadian-kids
# Header Mobile Buttons

## Notion Ticket Link

[Notion Ticket](https://www.notion.so/uwblueprintexecs/Make-header-bar-mobile-friendly-0a10cf7f46ce40df9d14da5116396932)



## Implementation Description
* Added dropdown style menu with settings and logout button for mobile screen sizes.



## Steps To Test
1. Test to make sure button UI and functionality for mobile view works as expected.



## What Should Reviewers Focus On?
* Is there a better way to implement the logout mobile button without having to move a lot of the logout functionality into the header file.


## Checklist
